rust/helm/hyperlane-agent/templates/scraper-statefulset.yaml

```diff
@@ -53,11 +53,20 @@ spec:
         - secretRef:
             name: {{ include "agent-common.fullname" . }}-secret
         - secretRef:
-            name: {{ include "agent-common.fullname" . }}-scraper-secret
+            name: {{ include "agent-common.fullname" . }}-scraper3-secret
         env:
+#{{- $scraperChainNames := list }}
+#{{- range .Values.hyperlane.chains }}
+#{{- if not .disabled }}
+#{{- $scraperChainNames = append $scraperChainNames .name }}
+#{{- end }}
+#{{- end }}
+          - name: HYP_SCRAPER_CHAINSTOSCRAPE
+#            value: {{ $scraperChainNames | join "." }}
```

Comment:
Before this gets merged we need to update the helm deployment to include all chains.

Reply:
Need any help here?